src/main/resources/application.yaml

```
@@ -0,0 +1,20 @@
+    username: admin
+    password: rbflchlrh12
+    driver-class-name: com.mysql.cj.jdbc.Driver
+
+  jpa:
+    hibernate:
+      ddl-auto: update   # 개발용: 스키마 자동 생성/업데이트 (운영에선 validate 권장)
+    show-sql: true
+    properties:
+      hibernate:
+        format_sql: true
+        dialect: org.hibernate.dialect.MySQL8Dialect  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)